src/main/java/org/example/pages/RegistrationPage.java

```diff
@@ -36,4 +45,75 @@ public RegistrationPage clickRegistrationButton() {
         getDriver().findElement(REGISTRATION_BUTTON_LOCATOR).click();
         return this;
     }
+
+    public RegistrationPage checkUsernameFieldIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(USERNAME_INPUT_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage checkPasswordFieldIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(PASSWORD_INPUT_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage checkRepeatPasswordFieldIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(REPEAT_PASSWORD_INPUT_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage checkRegistrationButtonIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(REGISTRATION_BUTTON_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage checkBackToLoginPageLinkIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(ALREADY_HAVE_AN_ACCOUNT_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage checkCreateAUserTitleIsDisplayed() {
+        Assertions.assertTrue(getDriver().findElement(CREATE_A_USER_TITLE_LOCATOR).isDisplayed());
+        return this;
+    }
+
+    public RegistrationPage check150OverCharactersError() {
```

Comment:
I'd call it TooLongValueError or something like that. It's generally considered as a bad practice to use digits in names.